src/DisputeManager.ts

```diff
@@ -208,10 +198,34 @@ function loadOrCreateAppeal(disputeId: BigInt, roundNumber: BigInt, event: Ether
   return appeal
 }
 
+export function createJurorDraft(disputeManagerAddress: Address, disputeId: BigInt, roundId: BigInt, jurorAddress: Address, timestamp: BigInt): JurorDraft {
+  let manager = DisputeManager.bind(disputeManagerAddress)
+  let response = manager.getJuror(disputeId, roundId, jurorAddress)
+  let disputeRoundId = buildRoundId(disputeId, roundId)
+  let draftId = buildDraftId(disputeRoundId, jurorAddress)
+  let draft = new JurorDraft(draftId)
+  draft.round = roundId.toString()
```

Comment:
Actually this should be `disputeRoundId.toString()` 😄

Reply:
Oh, good catch!! Thanks!!